assets/dev/scss/editor/preview/preview.scss

```diff
@@ -242,6 +242,11 @@ html.elementor-html {
 			text-align: center;
```

Comment:
Originally I had made these updates originally inside the packages repository, but Evyatar asked me to update this in the existing stylesheets.